packages/terra-date-time-picker/src/DateTimePicker.jsx

```diff
@@ -335,11 +344,20 @@ class DateTimePicker extends React.Component {
 
     if (newDateTime.isDST()) {
       newDateTime.add(1, 'hour');
-      this.handleChange(event, newDateTime);
+      this.setState({
+        dateTime: newDateTime,
+      });
+      if (this.props.onChange) {
+        this.props.onChange(event, newDateTime && newDateTime.isValid() ? newDateTime.format() : '');
+      }
+    } else if (this.props.onChange && !this.wasOffsetButtonClicked) {
```

Comment:
It might be helpful to add a comment to explain the workflow and why we're making the check on wasOffsetButtonClicked. I know I will forget several months later.

Reply:
Added in https://github.com/cerner/terra-framework/pull/555/commits/105f1cae0c24d88d4c2096cf5bd7c3fd28a755ec.